recordium/utils.py

```diff
@@ -26,11 +25,22 @@ def _ensure_dir_exists(dirpath):
 
 
 # simple attributes to have this calculated at one place only
-config_basedir = BaseDirectory.xdg_config_home
-cache_basedir = os.path.join(BaseDirectory.xdg_cache_home, 'recordium')
-_ensure_dir_exists(cache_basedir)
-data_basedir = os.path.join(BaseDirectory.xdg_data_home, 'recordium')
-_ensure_dir_exists(data_basedir)
+app = appdirs.AppDirs("recordium")
+
+# if exists, move the old config file into a new dir configuration.
+if not os.path.exists(app.user_config_dir):
+    config_file_name = "recordium.cfg"
+    _ensure_dir_exists(app.user_config_dir)
+    prev_cfg = os.path.join(os.path.split(app.user_config_dir)[0], config_file_name)
```

Comment:
The `os.path.split(...)[0]` is simpler and less confusing doing `os.path.dirname(app.user_config_dir)`